mslib/msui/ui/ui_mscolab_connect_dialog.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>478</width>
-    <height>255</height>
+    <height>268</height>
```

Comment:
The "Login by Identity Provider" button is slightly cutoff on my system. See:

![Screenshot from 2023-07-14 12-08-44](https://github.com/Open-MSS/MSS/assets/9308656/addbb3f9-45d3-4e51-8fa0-02e936f1feda)

Is this happening only for me?

Weirdly if I increase the height value to 270 it is no longer cut off, but if I increase it to 272 it is cut off again. Seems like the vertical layout behaves weird with regards to automatically filling the available space. This might be caused by me using material-shell instead of some more common DE, so maybe this is a non-issue.

Reply:
it works for me on both 268, and 272..

This is how 272 appears for me.
![image](https://github.com/Open-MSS/MSS/assets/57173445/27f5ccef-78b5-4c5b-bbc4-1a767127d18b)

However i set it to 270.